week1/citibike.sh

```diff
@@ -4,20 +4,34 @@
 #
 
 # count the number of unique stations
+cut -d, -f4 201402-citibike-tripdata.csv | sort | uniq | head -n-1 | wc -l
 
 # count the number of unique bikes
+cut -d, -f12 201402-citibike-tripdata.csv | sort | uniq | head -n-1 | wc -l
 
 # count the number of trips per day
+cut -d, -f2 201402-citibike-tripdata.csv | cut -d' ' -f1  | sort | uniq -c | head -n-1
 
 # find the day with the most rides
+cut -d, -f2 201402-citibike-tripdata.csv | cut -d' ' -f1 | uniq -c |sort -nr | head -n1
+ 
 
 # find the day with the fewest rides
+cut -d, -f2 201402-citibike-tripdata.csv | cut -d' ' -f1 | uniq -c | sort | tail -n+2 | head -n1
```

Comment:
Cool! I actually had to look up what the + does in tail. I'll have to use that in the future.

Reply:
Thank you! Found out a suggestion with head -n-1 so used the same idea for tail.